doc/troubleshooting.md

```diff
@@ -106,7 +106,7 @@ pravega-zk-headless                     ClusterIP      None             <none>
 
 ```
 
-However this problem has been solved in Kubectl client version `v1.11.x` onwards.
+However this problem has been solved in kubectl client version `v1.11.x` onwards.
```

Comment:
Please mention specific version number with which this was tested, instead of `v1.11.x`. Issue mentions this being fixed in `v1.12.0`

Reply:
Updated